frontend/src/metabase/core/components/Button/Button.tsx

```diff
@@ -84,11 +87,14 @@ const BaseButton = forwardRef(function BaseButton(
         )}
         {children && (
           <div
```

Comment:
We could make this a styled component and add to other exported parts, e.g. `Root`:

```
export default Object.assign(Button, {
  Root: ButtonRoot
});
```

The current solution with `contentClassName` makes it very hard to be used with `styled`

Reply:
@ranquild done, thanks, it became much better 🚀